### ISSUE TITLE
Unable to build on Fedora

### ISSUE THREAD

**Issue by fabaff** [jelmer/subvertpy#26]
It seems that we are no longer able to build the package on Fedora due to a very large amount of errors and warning.

```
subvertpy/_ra.c:2547:2: warning: initialization makes integer from pointer without a cast [-Wint-conversion]
subvertpy/_ra.c:2509:2: warning: initialization from incompatible pointer type [-Wincompatible-pointer-types]
subvertpy/_ra.c:2369:45: error: 'CredentialsIter_Type' undeclared (first use in this function)
subvertpy/util.c:589:17: warning: implicit declaration of function 'PyString_AS_STRING' [-Wimplicit-function-declaration]
```

**Comment by jelmer** [comment]
Are you building against python2 or python3?

**Comment by fabaff** [comment]
Both. Running with the default options for `gcc` on the build system for the RPM packages.

**Comment by jelmer** [comment]
Can you paste the full build logs somewhere? E.g. PyString_AS_STRING still exists in the latest versins of Python2, so that should build fine.

**Comment by fabaff** [comment]
https://kojipkgs.fedoraproject.org//work/tasks/3078/17433078/build.log

**Comment by jelmer** [comment]
On Fri, Jan 27, 2017 at 01:13:39AM -0800, Fabian Affolter wrote:
> https://kojipkgs.fedoraproject.org//work/tasks/3078/17433078/build.log
You are building against Python 3, which the master version of subvertpy does not support
(there is a separate branch that does).

Jelmer

**Comment by fabaff** [comment]
I just looked at the `setup.py` in `master` and found the Python 3.4 entry.

**Comment by jelmer** [comment]
Ah, oops :( I've removed that label for now. Work on python3 support is in the python3-branch and python3-merge branches

**Comment by jelmer** [comment]
Python3 support has been merged, fixed in 0.10.0.
